reference/artifacts/in-pipelines/index.md

```diff
@@ -38,12 +38,19 @@ artifact.
 The matching behavior is as follows: The artifact to match declared by the
 expected artifact has the same format as a regular artifact; however, its
 values are interpreted as regular expressions to match the corresponding values
-in the incoming artifact.
+in the incoming artifact:
+- Normal strings must match exactly for the artifact to match because Spinnaker
+effectively treats them as regular expressions. For example, Spinnaker
+interprets `inputstring` as `^inputstring$`.
+- Explicit regular expressions are also valid. For example, Spinnaker will
```

Comment:
```suggestion
- Explicit regular expressions are also valid. For example, Spinnaker
```

Reply:
Great call, thanks for the speedy review!